Upstream: Yes, not yet committed
Source: Ingmar Vanhassel <ingmar@exherbo.org>
Reason: Fails to build with gcc-4.4

From ec121d2e9bca8d18a819ca630d0feb11ae0331e1 Mon Sep 17 00:00:00 2001
From: Bernd Steinhauser <berniyh@exherbo.org>
Date: Sun, 28 Jun 2009 11:31:55 +0000
Subject: [PATCH] GCC 4.4.0 fixes

---
 plugins/yahoo/yahooauth.cpp   |    1 +
 plugins/yahoo/yahoosearch.cpp |    2 ++
 sim/event.cpp                 |    1 +
 3 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/plugins/yahoo/yahooauth.cpp b/plugins/yahoo/yahooauth.cpp
index d57581c..19f17fe 100644
--- a/plugins/yahoo/yahooauth.cpp
+++ b/plugins/yahoo/yahooauth.cpp
@@ -49,6 +49,7 @@
 #include <openssl/md5.h>
 
 #include <ctype.h>
+#include <cstdio>
 //Added by qt3to4:
 #include <Q3CString>
 
diff --git a/plugins/yahoo/yahoosearch.cpp b/plugins/yahoo/yahoosearch.cpp
index 1f14c89..8e825ad 100644
--- a/plugins/yahoo/yahoosearch.cpp
+++ b/plugins/yahoo/yahoosearch.cpp
@@ -21,6 +21,8 @@
 #include <Q3CString>
 #include <QShowEvent>
 
+#include <cstdio>
+
 #include "country.h"
 #include "intedit.h"
 #include "log.h"
diff --git a/sim/event.cpp b/sim/event.cpp
index 2cae99d..c33f4f0 100644
--- a/sim/event.cpp
+++ b/sim/event.cpp
@@ -19,6 +19,7 @@
 #include <QMultiMap>
 #include <stdio.h>
 #include <time.h>
+#include <cstdio>
 // own header
 #include "event.h"
 // local includes
-- 
1.6.3.3

